src/metabase/pulse/render/body.clj

```diff
@@ -795,12 +795,15 @@
         render-fn      (if (isa? (-> cols x-axis-rowfn :effective_type) :type/Temporal)
                          js-svg/timelineseries-waterfall
                          js-svg/categorical-waterfall)
+        show-total     (if (nil? (:waterfall.show_total viz-settings))
+                         true
+                         (:waterfall.show_total viz-settings))
```

Comment:
This is just making `true` the default, right? You could just do `(:waterfall.show_total viz-settings true)`

Reply:
I've seen `nil` as actual set value in it once or twice, for which this semantics is different, but that's legit another bug, too, lol